src/entities/coach/playbook/default/playbook_default.cpp

```diff
@@ -0,0 +1,51 @@
+/***
+ * Maracatronics Robotics
+ * Federal University of Pernambuco (UFPE) at Recife
+ * http://www.maracatronics.com/
+ *
+ * This file is part of Armorial project.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ ***/
+
+#include "playbook_default.h"
+
+Playbook_Default::Playbook_Default() {
+
+}
+
+QString Playbook_Default::name() {
+    return "Playbook_Default";
+}
+
+void Playbook_Default::configure(int numPlayers) {
```

Comment:
In the playbook **configure()** and **run()** method you have the _numPlayers_ param, so you can use it to configure the exactly quantity of roles that you need, and **configure()** is called every time that the number of players are changed in the field.